CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ IF(COMMAND CMAKE_POLICY)
 	CMAKE_POLICY(SET CMP0005 NEW)
 	CMAKE_POLICY(SET CMP0003 NEW)
 	IF (CMAKE_MAJOR_VERSION GREATER 2)
-		CMAKE_POLICY(SET CMP0026 OLD)
+		CMAKE_POLICY(SET CMP0026 NEW)
```

Comment:
Can we just remove it entirely?

Reply:
No idea... maybe @lukas-w can tell. I don't know anything about cmake, I just opened this PR because in #3985 @lukas-w said it was safe to set it to `NEW` while remove the warning.